maven-core/src/main/java/org/apache/maven/plugin/internal/DefaultPluginValidationManager.java

```diff
@@ -70,8 +72,12 @@ private ValidationLevel validationLevel(RepositorySystemSession session) {
         try {
             return ValidationLevel.valueOf(level.toUpperCase(Locale.ENGLISH));
         } catch (IllegalArgumentException e) {
-            throw new IllegalArgumentException(
-                    "Invalid value specified for property " + MAVEN_PLUGIN_VALIDATION_KEY + ": '" + level + "'", e);
+            logger.warn(
+                    "Invalid value specified for property '{}': '{}'. Supported values are (case insensitive): {}",
```

Comment:
The single quotes aren't necessary after then colon. That's the purpose of the colon.

Reply:
You meant _before_ the column ?

Reply:
No, I expect the statement to be: `"Invalid value specified for property '{}': {}. Supported values are (case insensitive): {}"`

Reply:
Ok, that was the exact opposite before this PR.  I would assume quotes are more needed for a user supplied value (which could be empty, with spaces, etc...) rather than for the property name which we do control.

Reply:
Same here: changed to quoted user supplied value (that really can be anything)

Reply:
OK. let's leave the change. I am fine with the concensus!